frontend/package.json

```diff
@@ -54,24 +55,25 @@
     "lodash-es": "^4.17.15",
     "node-gyp": "^7.1.2",
     "node-sass": "^4.14.1",
-    "react": "^17.0.2",
+    "react": "^18.2.0",
     "react-cool-dimensions": "^2.0.5",
-    "react-dom": "^17.0.2",
-    "react-redux": "^7.2.1",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.0.4",
     "react-router": "^6.4.1",
     "react-router-dom": "^6.4.1",
-    "redux": "^4.0.5",
```

Comment:
Did you remove redux or am I blind?

Reply:
yeah i removed it. https://redux.js.org/redux-toolkit/overview#purpose. I mentioned this in the description. It's lightweight and contains everything we need already. It's the same and officially maintained.

Reply:
I don't understand... our goal is to get rid of redux in the long run -- why would you change our backing library?

Reply:
I can bring it back if you want, actually it makes no difference, we are not heavily using redux in our application so we only used a very small part of functions in it.